Dockerfile

```diff
@@ -32,8 +32,12 @@ RUN \
 
 FROM alpine:3.9
 
+EXPOSE 22124/udp
```

Comment:
The port can be specified from command line. I think it's better to be explicit and expose it from command line.

Reply:
The docker documentation states:

> The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.

So maybe EXPOSE would make sense, then?

Reply:
Having the EXPOSE line here makes it easier to discover which port will be used externally. A good reason to have it here IMO.